VSCode/armsnippets.json

```diff
@@ -15,6 +15,25 @@
     ],
     "description": "Skeleton ARM Template"
 },
+"Azure Resource Manager (ARM) Variable": {
```

Comment:
I'd name these snippets simply "Variable" and "Parameter" (no "Azure Resource Manager (ARM) " prefix needed)